### PR TITLE
bump 0.27

### PR DESCRIPTION
**Pull request by eliegaboriau**
This pr enables the module to be used with 0.27 decidim.

It bumps :
- the decidim version
- the ruby version

It updates the rakefile (to fix the seed in development app problem)
It updates the import css file to remove error.

The rest of the changes are due to the linter

### CODE REVIEW

**Comment by paulinebessoles** [comment]
Hi @eliegaboriau! Thank you for the PR 😁 Could you also save a branch for 0.26 version, so we can continue using ludens on our applications?

File: app/models/decidim/ludens/participative_action.rb [hunk 1]
```diff
@@ -9,8 +9,6 @@ class ParticipativeAction < ApplicationRecord
                  foreign_key: "decidim_organization_id",
                  class_name: "Decidim::Organization"
 
-      validates :organization, presence: true
```
Comment (armandfardeau) [inline app/models/decidim/ludens/participative_action.rb:12]: Are you sure you want to delete the validation?

Comment (eliegaboriau) [reply app/models/decidim/ludens/participative_action.rb:12]: Nope it's an error i'm reverting that

Comment (eliegaboriau) [reply app/models/decidim/ludens/participative_action.rb:12]: Actually it's not a mistake, it's a correction by rubocop : it's already defined by the belongs_to attribute so it's redondant to add the validation

Comment (armandfardeau) [reply app/models/decidim/ludens/participative_action.rb:12]: Ok !

**Review by armandfardeau — APPROVED** [review]

**Review by paulinebessoles — APPROVED** [review]

**Pull request merged**